braindecode/classifier.py

```diff
@@ -318,4 +318,5 @@ def predict_trials(self, X, return_targets=True):
             module=self.module,
             dataset=X,
             return_targets=return_targets,
+            num_workers=self.iterator_valid__num_workers,
```

Comment:
`iterator_valid__num_workers` may not exist, so 
```suggestion
            getattr(clf, 'iterator_valid__num_workers', 0),
```
or 
```suggestion
            clf.get_iterator(X, training=False).loader.num_workers,
```

Reply:
Slightly prefer the second as I feel scared the first one will always return 0 even if variable names have changed etc... so even iif sth goes wrong we wouldn't notice